views/project/billing.erb

```diff
@@ -301,3 +301,108 @@
     ) %>
   </form>
 <% end %>
+<br/>
+<!-- Usage alerts -->
+<div>
+  <div class="md:flex md:items-center md:justify-between pb-1 lg:pb-2">
+    <div class="min-w-0 flex-1">
+      <h3 class="text-2xl font-bold leading-7 text-gray-900 sm:truncate sm:text-2xl sm:tracking-tight">
+        Usage Alerts
+      </h3>
+    </div>
+  </div>
+  <div class="grid gap-6">
+    <div class="overflow-hidden rounded-lg shadow ring-1 ring-black ring-opacity-5 bg-white divide-y divide-gray-200">
+      <!-- Add usage alert -->
+      <div class="px-4 py-5 sm:p-6">
+        <form action="<%= "#{@project_data[:path]}/usage-alert" %>" role="form" method="POST">
+          <%== csrf_tag("#{@project_data[:path]}/usage-alert") %>
+          <div class="space-y-4">
+            <div>
+              <h2 class="text-lg font-medium leading-6 text-gray-900">Add new usage alert</h2>
+              <p class="mt-1 text-sm text-gray-500">
+                Usage alerts will be sent to your email address. If you want to send alerts to specific email
+                addresses, you need to log in with that email address.
+                <br/>
+                <br/>
+                Please note that alerts are only for informational purposes and no action is taken automatically.
+              </p>
+            </div>
+            <div class="grid grid-cols-12 gap-6">
+              <div class="col-span-5 sm:col-span-3">
+                <%== render(
+                  "components/form/text",
+                  locals: {
+                    name: "alert_name",
+                    label: "Alert Name",
+                    type: "text",
+                    attributes: {
+                      required: true,
+                      placeholder: "Alert Name"
+                    },
+                    extra_class: "pr-3"
+                  }
+                ) %>
+              </div>
+              <div class="col-span-5 sm:col-span-3">
+                <%== render(
+                  "components/form/text",
+                  locals: {
+                    name: "limit",
+                    type: "number",
+                    attributes: {
+                      required: true,
+                      placeholder: "100"
+                    },
+                    extra_class: "pr-3"
+                  }
+                ) %>
+              </div>
```

Comment:
Can you add a label to limit too or remove the label from name? They look unaligned vertically.